apps/authentication-portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -312,6 +312,16 @@
         <jsp-file>/totpError.jsp</jsp-file>
     </servlet>
 
+    <servlet>
+        <servlet-name>backup_code.do</servlet-name>
+        <jsp-file>/backupCode.jsp</jsp-file>
+    </servlet>
+
+    <servlet>
+        <servlet-name>backup_code_error.do</servlet-name>
+        <jsp-file>/backupCodeError.jsp</jsp-file>
```

Comment:
Verify the file name convention. AFAIR, we have `-` separated file names.

Reply:
In some places we used `-` but in some places we used camel case. But most of the cases, `-` is used. So I'll update the file names with `-`